core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -3803,6 +3803,33 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-ibmcloud
     namespace: ci
+- from:
```

Comment:
Before develop install work, you need to get cluster profile get ready in advance, can not do that in one PR.